### DISCUSSION TITLE
Trouble Installing Weblate Locally: Errors during "pip install Weblate[all]"

### DISCUSSION THREAD

**Discussion by sanimkhan** [WeblateOrg/weblate#9780] [General]
Hello everyone,

I'm currently facing some hurdles while attempting to set up Weblate locally on my system. I've meticulously followed the provided guidelines from the Weblate documentation, including the steps outlined in the following links:

[Contributing to Weblate](https://docs.weblate.org/en/latest/contributing/start.html)
[Installing Weblate from Source](https://docs.weblate.org/en/latest/admin/install/source.html)
[Virtual Environment Installation on macOS](https://docs.weblate.org/en/latest/admin/install/venv-macos.html)

However, I've encountered an issue when executing the command pip install "Weblate[all]". The error details are as follows:

**Error 1:**

```
Building wheel for misaka (setup.py) ... error
error: subprocess-exited-with-error
  Building wheel for misaka (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [42 lines of output]
      /Users/sanimkhan/weblate-env/lib/python3.10/site-packages/setuptools/__init__.py:84: _DeprecatedInstaller: setuptools.installer and fetch_build_eggs are deprecated.
      !!
```

**Error 2:**

```
      <scratch space>:205:1: note: expanded from here
      xmlSecSoap12Ns
      ^
      /opt/homebrew/Cellar/libxmlsec1/1.3.1_1/include/xmlsec1/xmlsec/strings.h:34:33: note: 'xmlSecXPath2Ns' declared here
      XMLSEC_EXPORT_VAR const xmlChar xmlSecXPath2Ns[];
                                      ^
      2 errors generated.
      error: command '/usr/bin/clang' failed with exit code 1```

**Comment by nijel** [comment]
See https://github.com/xmlsec/python-xmlsec/issues/254 for possible workarounds
